<launch>
  <param name="data_path" type="string" value="/data/bs_hall/spot0/fullview_recon/global_registered_cloud.pcd" />
  <param name="save_path" type="string" value="/data/bs_hall/spot0/fullview_recon/segment.pcd" />
  <!-- mode = 0: delete points inside the polygon -->
  <!-- mode = 1: save points inside the polygon -->
  <param name="clip_mode" type="bool" value="0" />
  <param name="k_nearest" type="int" value="10" />
  <node name="segment" pkg="calibration" type="segment" output="screen">
  </node>
</launch>
